act-rules-format.md

```diff
@@ -206,6 +215,82 @@ For each accessibility requirement in the mapping, an ACT Rule <em class="rfc211
   </ul></blockquote>
 </aside>
 
+#### Secondary Requirements ####
+
+When both the "Failed Outcomes" and "Passed or Inapplicable Outcomes" described under [=Conformance Requirements=] are true for an accessibility requirement, the rule <em class="rfc2119">must</em> map that accessibility requirement as a Conformance Requirement.
+
+When only one of the Outcomes is true or both Outcomes are only sometimes true for an accessibility requirement, the rule <em class="rfc2119">should</em> map the accessibility requirement as Secondary. When an ACT rule maps to a Secondary requirement, it must include an explanation of why that requirement is Secondary in the Background section of the rule.
```

Comment:
I feel it is a bit hard to read because of "outcome" both being the outcome of a rule, and the items in the conformance requirement section 🤔 

```suggestion
When both the "Failed Outcomes" and "Passed or Inapplicable Outcomes" conditions described under [=Conformance Requirements=] are true for an accessibility requirement, the rule <em class="rfc2119">must</em> map that accessibility requirement as a Conformance Requirement.

When only one of these condition is true, or both conditions are only sometimes true for an accessibility requirement, the rule <em class="rfc2119">should</em> map the accessibility requirement as Secondary. When an ACT rule maps to a Secondary requirement, it must include an explanation of why that requirement is Secondary in the Background section of the rule.
```
Plus maybe making the conditions' name into anchor and linking back to them from here?